massgov/openmass
# DP-22180: Create reusable view for displaying collections content

**Description:**
Added a reusable Collections view, currently for the EOTSS Service Catalog.


**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-22180


**To Test:**
- Go to /collections/eotss-services.
- [ ] Verify the title is "EOTSS Service Catalog".
- [ ] Verify there is a "Filter by Topic" field with only topics.
- [ ] Verify there is a "Filter by Keywords" text field.
- [ ] Verify the results appear as expected.
- Go to /collections/eotss-services/desktop-software.
- [ ] Verify the title is "EOTSS Service Catalog: Desktop Software".
- [ ] Verify there is a "Filter by Sub Topic" field with only sub topics.
- Go to /collections/eotss-services/support-services.
- [ ] Verify there is NOT a "Filter by Sub Topic" field.
- Go to /data-listing/all.
- [ ] Verify it looks as expected.
-  Go to /collections/eotss-services/email-calendar-collaboration.
- [ ] Verify the title contains "&" and not "&amp;"
- Login as an Administrator or Content Administrator and go to /admin/structure/taxonomy/manage/collections/overview.
- Edit the "EOTSS Service Catalog" term.
- [ ] Verify the Landing Page link field help text is as expected.
- [ ] Verify the URL Name field help text is as expected.
- [ ] Verify the All Collection Breadcrumb Text field is as expected.


**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
